README.md

```diff
@@ -1,49 +1,182 @@
+
+#### PyCharm for macOS
+
+In PyCharm, open iit-backend using `File > Open`.
+
+When opening the project, PyCharm should detect the requirements.txt file and automatically prompt you to create a virtual environment for the project. The prompt should look like the picture below. Select your base interpreter and click `OK`. If this prompt doesn't appear, create a virtualenv interpreter manually [https://www.jetbrains.com/help/pycharm/project-interpreter.html#3b6542ac](https://www.jetbrains.com/help/pycharm/project-interpreter.html#3b6542ac)
+
+![Setup Prompt Window](pictures/setup_prompt.png "Setup Prompt Window")
+
+##### Installing Required Packages
+
+Using the steps above, PyCharm should automatically install all of the packages listed in `requirements.txt`. If you created your virtualenv manually or if PyCharm returns an error, try closing and reopening a `.py` file inside the project and click on the "install requirements" pop up. Alternatively, you can open the terminal in PyCharm and run `pip install -r requirements`.
+
+If you encounter any errors installing packages, try installing them manually via the terminal with no version requirements.
+
+##### Run Configuration
+
+In the menu bar, select `Run > Edit Configuration`.
+
+Click on the `+` symbol to add a new configuration and select Python. Configure as follows:
+
+- Name: Give your configuration any name
+- Script Path: select the script you want to run e.g. `manage.py` or `/app/github_public_repo/estimate_csv_creator.py`.
+- Parameters: `run`
+- Environment variables: Contact a team member for the complete list of environment variables
+- Python interpreter: select the python virtual environment you configured above
+- Interpreter options: leave blank
+- Working directory: This should automatically fill based on the `Script path` value
+- Add content roots to PYTHONPATH. Check this box
+- Add source roots to PYTHONPATH. Check this box
+
+Click OK.
+
+You should now be ready to run your script!
+
+#### PyCharm on Windows
+
+Note that this method can work using either native Windows or WSL. However, if you are using PyCharm, it is strongly recommended that you clone the repository within your **Windows user profile** and **NOT your WSL user profile**. PyCharm WSL compatibility is only available with [PyCharm Pro Edition](https://www.jetbrains.com/help/pycharm/using-wsl-as-a-remote-interpreter.html) and the workarounds for the Community Edition are not documented here. 
+
+##### Anaconda
+
+Install Anaconda [https://docs.anaconda.com/anaconda/install/windows/](https://docs.anaconda.com/anaconda/install/windows/)
+
+##### Creating virtual environment
+
+In PyCharm, open iit-backend using `File > Open`. Click *Cancel* if prompted to create a virtual environment.
+
+Create a virtualenv interpreter manually [https://www.jetbrains.com/help/pycharm/project-interpreter.html#3b6542ac](https://www.jetbrains.com/help/pycharm/project-interpreter.html#3b6542ac).
```

Comment:
Yeah I think because you have to create a conda environment you have to do it manually. I don't remember specifically which issues I ran into, but it definitely didn't work well. I could try this again sometime and update the documentation if I figure out anything. Or I could add more details on creating the environment manually instead of pointing to the guide.

Reply:
ya i would add something like u can try to let pycharm do it if you have issues make a condo env manually and here is how